cookiecutter-django-ida/{{cookiecutter.repo_name}}/Dockerfile

```diff
@@ -39,13 +39,16 @@ RUN locale-gen en_US.UTF-8
 ENV LANG en_US.UTF-8
 ENV LANGUAGE en_US:en
 ENV LC_ALL en_US.UTF-8
-ENV DJANGO_SETTINGS_MODULE {{cookiecutter.project_name}}.settings.production
 
 EXPOSE {{cookiecutter.port}}
 RUN useradd -m --shell /bin/false app
 
 WORKDIR /edx/app/{{cookiecutter.repo_name}}
 
+FROM app as prod
+
+ENV DJANGO_SETTINGS_MODULE {{cookiecutter.project_name}}.settings.production
+
 # Copy the requirements explicitly even though we copy everything below
 # this prevents the image cache from busting unless the dependencies have changed.
 COPY requirements/production.txt /edx/app/{{cookiecutter.repo_name}}/requirements/production.txt
```

Comment:
Does this instruction need to be duplicated for the dev target as well? Or is dev.txt getting copied somewhere else?

Reply:
Noted, added steps to copy `dev.txt` and the `code`